src/common/res/features/export-transactions/main.js

```diff
@@ -75,7 +75,23 @@
         downloadFile(args);
       }
 
+      // Sort an Array of Objects by key
+      function sortByKey(transactions, key, order) {
 
+        // Simple comparator
+        function compare(a, b) {
+          if (a[key] < b[key]) return -1;
+          if (a[key] > b[key]) return 1;
+          return 0;
+        }
+
+        transactions = transactions.copy();
+        transactions.sort(compare);
+        if (order === 'DESC') {
+          transactions.reverse
```

Comment:
In this case that particular function is just about sorting an array by a key, so I'm just looking at it going, "If you pass 'DESC' in it probably don't work as they'd expect'. Feel free to call it however you need to for your feature, I just want the function to work as it says it will. :+1: